src/v/storage/disk_log_impl.cc

```diff
@@ -551,10 +551,10 @@ ss::future<bool> disk_log_impl::sliding_window_compact(
           storage::internal::should_apply_delta_time_offset(_feature_table));
 
         vlog(
-          gclog.debug,
+          gclog.info,
```

Comment:
I know this is a cherry-pick from the v23.2 PR, but what was useful about this in debugging originally? I think this will be a bit verbose, and am thinking perhaps there are other places to log useful info

Reply:
_offset_tracker state from the segment output (this now logs the full segment info). The offset tracker tells whats the start/dirty offsets after compaction. 

> I think this will be a bit verbose, and am thinking perhaps there are other places to log useful info

I see.. I was on the fence about this but I thought it wouldn't too chatty. I don't mind lowering the log level, lmk.

Reply:
I thought a bit more, switched back to debug to be consistent with other places in this place.